src/logger.ts

```diff
@@ -2,8 +2,6 @@ import type { FastifyLoggerInstance } from "fastify";
 import { Counter } from "prom-client";
 import { inspect } from "util";
```

Comment:
Where can I see this import? I wanted to look at the inspect method but I don't know what library is `util`.

Reply:
`util` is nodejs builtin module: https://nodejs.org/dist/latest-v18.x/docs/api/util.html#utilinspectobject-options

Reply:
👌 perfect.

There is no way this will return an `undefined` or `null` value, right? Just because `formatted.replaceAll(secret, "[MASKED]");` would throw if `formatted` is null.

Reply:
`util.inspect()` returns string